server/accounts.go

```diff
@@ -694,6 +702,8 @@ func (a *Account) IsExportServiceTracking(service string) bool {
 // is the RTT used to calculate the total latency. The requestor's account can
 // designate to share the additional information in the service import.
 type ServiceLatency struct {
+	TypedEvent
+
 	Status         int           `json:"status"`
```

Comment:
Not for this PR, but should we make consistent with JSApi work? So `Error { Code: Description: }`?

Reply:
Yes, I was planning to do that when I add the `type` field for you - assuming you don't mind it this way

Reply:
ok yes consistency good ;)

We can do in separate PR.